attest/verifier/types/src/verification.rs

```diff
@@ -11,125 +17,23 @@ use mc_crypto_keys::X25519Public;
 use mc_sgx_core_types::QuoteNonce;
 use mc_sgx_dcap_types::{Collateral, Quote3};
 use mc_util_encodings::{Error as EncodingError, FromBase64, FromHex};
-use prost::{
-    bytes::{Buf, BufMut},
-    encoding::{self, DecodeContext, WireType},
-    DecodeError, Message, Oneof,
-};
 use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
 
-#[derive(Clone, Debug, Default, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
```

Comment:
If so desired we could use the `prost::DcapEvidence` everywhere and not need this type. It would put us more or less where we were.  I like having the type with non optional fields, but I'm fine seeing the other approach too.

Reply:
I like the non-optional fields as well.